plaso/parsers/sqlite_plugins/dropbox.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for Dropbox sync_history database files."""
+
+from dfdatetime import posix_time as dfdatetime_posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class DropboxSyncHistoryEventData(events.EventData):
+  """Dropbox Sync History Database event data.
+
+  Attributes:
+    event_type (str): the event type
+    file_event_type (str): the file event type
+    direction (str): the source of the synchronisation event
+    file_id (str): the ID of the file.
+    local_path (str): the local path of the file.
+  """
+
+  DATA_TYPE = 'dropbox:sync_history:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(DropboxSyncHistoryEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.event_type = None
+    self.file_event_type = None
+    self.direction = None
+    self.file_id = None
+    self.local_path = None
+
+
+class DropboxSyncDatabasePlugin(interface.SQLitePlugin):
+  """SQLite parser plugin for Dropbox sync_history.db database files.
+
+  The Linux sync_history.db database is typically stored in:
```

Comment:
I opt to put this into an artifact definition, if not already

Reply:
Ack, already in the works.  Will submit a PR for the updated definition.